### PR TITLE
AN-4539 mints and update sales - fsc_utils upgrade

### PR DESCRIPTION
**Pull request by eric-laurello**

### CODE REVIEW

File: models/silver/nft/mints/silver__nft_mints_combined.sql [hunk 1]
```diff
@@ -0,0 +1,92 @@
+{{ config(
+    materialized = 'incremental',
+    unique_key = "nft_mints_combined_id",
+    incremental_strategy = 'merge',
+    cluster_by = ['block_timestamp::DATE','_inserted_timestamp::DATE'],
+    merge_exclude_columns = ["inserted_timestamp"],
+    tags = ['noncore']
+) }}
+
```
Comment (tarikceric) [inline models/silver/nft/mints/silver__nft_mints_combined.sql:9]: some instances where there seems to be multiple tokens in the `currency_address`:

```
select * from aptos_dev.silver.nft_mints_combined
        where currency_address = '0xc7efb4076dbe143cbcd98cfaaa929ecfc8f299203dfff63b95ccb6bfe19850fa::swap::LPToken<0x1::aptos_coin::AptosCoin, 0x7ffddc750d48072559a661caea2342a940bbc7df4553b322f08891a760f67822::BNT::BNT' 
        limit 5;
```

File: models/gold/nft/nft__ez_nft_mints.sql [hunk 2]
```diff
@@ -0,0 +1,77 @@
+{{ config(
+    materialized = 'view',
+    tags = ['noncore'],
+    meta ={ 'database_tags':{ 'table':{ 'PURPOSE': 'NFT' }} }
+) }}
+
```
Comment (tarikceric) [inline models/gold/nft/nft__ez_nft_mints.sql:6]: more of an issue with the prices table, but a lot of total_price_usd is null for tokens that i assumed we would have like '0xf22bede237a07e121b56d91a491eb7bcdfd1f5907926a9e58338f964a01b17fa::asset::USDC'

File: models/silver/nft/mints/silver__nft_mints_combined.sql (hunk 1 above)
Comment (eric-laurello) [reply models/silver/nft/mints/silver__nft_mints_combined.sql:9]: yeah looks to be some duplication too on the mint side - will look into mitigating

File: models/gold/nft/nft__ez_nft_mints.sql (hunk 2 above)
Comment (eric-laurello) [reply models/gold/nft/nft__ez_nft_mints.sql:6]: mostly due to us not having prices before 11/28/23 -- going to look into why/if I can backfill further

File: models/silver/nft/mints/silver__nft_mints_combined.sql (hunk 1 above)
Comment (eric-laurello) [reply models/silver/nft/mints/silver__nft_mints_combined.sql:9]: I think we're good here. Some of these seem to mint the same token twice. See https://aptoscan.com/transaction/99262891

The currency is weird, but that's coming right from the transfer. Not directly related to this mint logic itself.

**Review by tarikceric — APPROVED** [review]

**Pull request merged**